fiftyone/core/collections.py

```diff
@@ -3212,6 +3213,243 @@ def _set_labels(self, field_name, sample_ids, label_docs, progress=False):
     def _delete_labels(self, ids, fields=None):
         self._dataset.delete_labels(ids=ids, fields=fields)
 
+    def update_samples(
```

Comment:
Added an `update_samples()` method for the case where the user just wants to apply edits to a collection.